packages/help-center/src/components/help-center-third-party-cookies-notice.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-restricted-imports */
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useEffect } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { getSectionName } from 'calypso/state/ui/selectors';
+import { BackButton } from './back-button';
+
+const ThirdPartyCookiesNotice: React.FC = () => {
+	const { __ } = useI18n();
+	const sectionName = useSelector( getSectionName );
+
+	useEffect( () =>
+		recordTracksEvent( 'calypso_helpcenter_third_party_cookies_notice_open', {
+			force_site_id: true,
+			location: 'help-center',
+			section: sectionName,
+		} )
+	);
```

Comment:
This will run many times, needs an empty dep array.

Reply:
Very nice catch! Fixed in https://github.com/Automattic/wp-calypso/pull/76777/commits/852ace6f1fc649f5c48b6cc730a1b64d6ec2cf2a. Technically, the effect depends on `sectionName`, but I've kept the dependencies empty (`[]`) so that it only fires on mount. Otherwise, it'd fire every time when the notice is on screen and the user is changing sections.